backend/src/main/java/com/now/naaga/player/presentation/dto/PlayerResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.now.naaga.player.presentation.dto;
+
+import com.now.naaga.player.domain.Player;
+
+public class PlayerResponse {
+
+    private Long id;
+    private String nickname;
+    private int totalScore;
+
+    public PlayerResponse() {
+    }
+
+    public PlayerResponse(final Long id,
+                          final String nickname,
+                          final int totalScore) {
+        this.id = id;
+        this.nickname = nickname;
+        this.totalScore = totalScore;
+    }
+
+    public static PlayerResponse to(final Player player) {
```

Comment:
```suggestion
    public static PlayerResponse from(final Player player) {
```
to 라는 메서드명은 PlayerResponse가 Player로 바뀌는 메서드를 뜻하는 것 같아요. 이 경우에는 from이 적절할 것 같습니다.